tests/unit/util/test_log_data_creator.py

```diff
@@ -0,0 +1,155 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""Unit tests for the log data creator utility module."""
+
+from datetime import datetime
+from typing import Any
+
+import pytest
+from starlette.requests import Request
+from starlette.types import Message
+
+from merino.middleware import ScopeKey
+from merino.middleware.geolocation import Location
+from merino.middleware.user_agent import UserAgent
+from merino.util.log_data_creators import (
+    create_request_summary_log_data,
+    create_suggest_log_data,
+)
+
+
+@pytest.mark.parametrize(
+    "headers,expected_agent,expected_lang",
```

Comment:
It's a personal preference, but I find the following syntax more readable:

```suggestion
    ["headers", "expected_agent", "expected_lang"],
```

Reply:
Change applied: [9162333](https://github.com/mozilla-services/merino-py/pull/106/commits/91623339988fd4e41fb20b4beef14ba3f6edf2fc)